include/linux/blkdev_compat.h

```diff
@@ -351,8 +351,8 @@ bio_set_flush(struct bio *bio)
 	bio_set_op_attrs(bio, 0, REQ_PREFLUSH);
 #else
 #error	"Allowing the build will cause bio_set_flush requests to be ignored."
-	"Please file an issue report at: "
-	"https://github.com/zfsonlinux/zfs/issues/new"
+#error	"Please file an issue report at: "
+#error	"https://github.com/zfsonlinux/zfs/issues/new"
```

Comment:
We should either use a multi-line macro for this, or drop the trailing 'Please file an issue' bit so it fits on one line.

```
#error  "Allowing the build will cause bio_set_flush requests to be ignored." \
        "Please file an issue report at: " \
        "https://github.com/zfsonlinux/zfs/issues/new"
```